webapp/src/main/webapp/iaf/gui/index.jsp

```diff
@@ -66,6 +66,7 @@ if(version.isEmpty()) {
 <script src="js/plugins/slimscroll/jquery.slimscroll.min.js"></script>
 <script data-pace-options='{ "ajax": false }' src="js/plugins/pace/pace-v1.2.4.min.js"></script>
 
+<script src="js/plugins/mermaid/mermaid.min.js"></script>
```

Comment:
Misschien dat het mooier is deze file te lazy loaden.. zou je kunnen kijken of dat mogelijk is? Er staan voorbeelden hoe andere pagina's het gebruiken in de config.js file.